_posts/2015-07-21-the-epages-selenium-framework.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "The ePages Selenium Framework"
+date: "2015-07-21 17:06:00"
+icon: ship
+tags: selenium, test, continuous delivery
+categories: tech-stories
+authors: ["Jan M.", "Martin K.", "Kay A."]
+---
+
+With the help of Selenium we create and run automated test scripts for
+the user interface of ePages. In order to make it quick and easy for
+the developers to write these tests we have built the ePages
+Selenium Framework.
+
+# Browser automation with Selenium
+
+[Selenium] is an open source project which provides a couple of tools
```

Comment:
This should be adapted if we mention in the beginning of the text what Selenium basically is (see my first comment).